src/main/java/fi/otavanopisto/kuntaapi/server/system/SystemHealthCheck.java

```diff
@@ -0,0 +1,56 @@
+package fi.otavanopisto.kuntaapi.server.system;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.logging.Logger;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.enterprise.context.Initialized;
+import javax.enterprise.event.Observes;
+import javax.enterprise.inject.Any;
+import javax.enterprise.inject.Instance;
+import javax.inject.Inject;
+
+import fi.otavanopisto.kuntaapi.server.cache.AbstractCache;
+
+@ApplicationScoped
+public class SystemHealthCheck {
+  
+  private static final String LINE = "------------------------------";
+  
+  @Inject
+  private Logger logger;
+
+  @Inject
+  @Any
+  private Instance<AbstractCache<?, ?>> caches;
+   
+  public void init(@Observes @Initialized(ApplicationScoped.class) Object init) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "init". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1172)